RockWeb/Themes/NewSpring/assets/Lava/DATA/get-permalink.lava

```diff
@@ -1,5 +1,6 @@
 {%- comment -%}Determine if this item's channel has a primary parent channel{%- endcomment -%}
 {% capture permalink %}
+{% assign cciid = cciid | AsInteger %}
```

Comment:
This change fixes an issue where a lava error was thrown because `cciid` was a string instead of an int.